bool data
geometry_msgs/Pose destination
---
bool success
geometry_msgs/Pose destination
